cmd/export_assets.go

```diff
@@ -18,86 +14,60 @@ var assetsCmd = &cobra.Command{
 	Long:  `Exports the assets that are created from payment operations over a specified ledger range`,
 	Run: func(cmd *cobra.Command, args []string) {
 		cmdLogger.SetLevel(logrus.InfoLevel)
-		endNum, useStdout, strictExport, isTest := utils.MustCommonFlags(cmd.Flags(), cmdLogger)
+		endNum, strictExport, isTest := utils.MustCommonFlags(cmd.Flags(), cmdLogger)
+		cmdLogger.StrictExport = strictExport
 		startNum, path, limit := utils.MustArchiveFlags(cmd.Flags(), cmdLogger)
+		gcsBucket, gcpCredentials := utils.MustGcsFlags(cmd.Flags(), cmdLogger)
+
+		outFile := mustOutFile(path)
 
 		paymentOps, err := input.GetPaymentOperations(startNum, endNum, limit, isTest)
 		if err != nil {
-			cmdLogger.Fatal("could not read assets: ", err)
-		}
-
-		var outFile *os.File
-		if !useStdout {
-			outFile = mustOutFile(path)
+			cmdLogger.Fatal("could not read asset: ", err)
 		}
 
 		// With seenIDs, the code doesn't export duplicate assets within a single export. Note that across exports, assets may be duplicated
 		seenIDs := map[uint64]bool{}
-		failures := 0
-		numBytes := 0
+		numFailures := 0
+		totalNumBytes := 0
 		for _, transformInput := range paymentOps {
 			transformed, err := transform.TransformAsset(transformInput.Operation, transformInput.OperationIndex, transformInput.Transaction, transformInput.LedgerSeqNum)
 			if err != nil {
 				txIndex := transformInput.Transaction.Index
-				errMsg := fmt.Sprintf("could not extract asset from operation %d in transaction %d in ledger %d: ", transformInput.OperationIndex, txIndex, transformInput.LedgerSeqNum)
-				if strictExport {
-					cmdLogger.Fatal(errMsg, err)
-				} else {
-					cmdLogger.Warning(errMsg, err)
-					failures++
-					continue
-				}
-
+				cmdLogger.Errorf("could not extract asset from operation %d in transaction %d in ledger %d: ", transformInput.OperationIndex, txIndex, transformInput.LedgerSeqNum)
```

Comment:
FYI the other command utilities use `LogErrror()` instead of `Errorf()` here

Reply:
Good catch. Updated.